src/ui/Quickview/Quickview.ts

```diff
@@ -250,7 +250,11 @@ export class Quickview extends Component {
 
     this.bindClick(result);
     if (this.bindings.resultElement) {
-      this.bind.on(this.bindings.resultElement, ResultListEvents.openQuickview, () => this.open());
+      this.bind.on(this.bindings.resultElement, ResultListEvents.openQuickview, (e: Event) => {
+        // Prevents parent result from opening quickview when in a folded result
+        e.stopPropagation();
```

Comment:
Would it be easy test that `stopPropagation` is called on a dummy event? I think it would be even better than leaving a comment, since when the test fails, we can read the test description to know what happened.

Reply:
I agree a test would be useful here